src/whylogs/core/statistics/constraints.py

```diff
@@ -318,23 +367,31 @@ def merge(self, other) -> "SummaryConstraint":
     @staticmethod
     def from_protobuf(msg: SummaryConstraintMsg) -> "SummaryConstraint":
 
-        if msg.HasField("value") and not msg.HasField("second_field") and not msg.HasField("between"):
+        if msg.HasField("reference_set") and not msg.HasField("value") and not msg.HasField("second_field") and not msg.HasField("between"):
```

Comment:
`if msg.HasField("reference_set") and not any([msg.HasField(f) for f in ["value", "second_field", "between"]]):`